resources/js/components/pages/Collections.vue

```diff
@@ -283,6 +332,7 @@ const loadMoreCollectionsWithObserver = () => {
                     const res = await CollectionApi.getCollections(collections.value.cursor);
                     const data = DTOFactory.forCollections(res);
```

Comment:
Can't we reuse the data here to generate the names? so we don't need to do an extra api calls

Reply:
The attribute value was parsed in the BE already https://github.com/enjin/platform-core/blob/master/src/Models/Laravel/Attribute.php#L54

Reply:
if there is URI, i dont think we can get it from somewhere else, does platform provide collection metadata ?

Reply:
Right, don't think its being parsed in the BE